tools/sgen/input/FluentUIStyle.yml

```diff
@@ -137,6 +137,46 @@ Opacity:
   opacity96: 0.96f
   none: 1f
 
+AP_AvatarTokens:
+  backgroundDefaultColor: "#EDACB1"
```

Comment:
this can't be right..... default color should be app accent color?

Reply:
Following up with Victor there. The figma file indicates that we'll have new colors for standard fallback image and background colors.

Reply:
Tracking this with #340